README

```diff
@@ -18,7 +18,7 @@ Steps to update pod spec (should be done per release):
   - AdobeMobileLibrary_Watch.a -> libAdobeMobile_Watch.a
   - AdobeMobileLibrary_TV.a -> libAdobeMobile_TV.a
   - AdobeMobileLibrary_Extension.a -> libAdobeMobile_Extension.a
-2. Update the SDK version number in AdobeMobileSDK.podspec (update s.version and s.source tag)
+2. Update the SDK version number in AdobeMobileSDK.podspec (update s.version tag)
```

Comment:
haha, i made the change to not have to fix the source tag last release but didn't update the README! :D